packages/service-library/src/servicelib/fastapi/postgres_lifespan.py

```diff
@@ -23,7 +23,7 @@ class PostgresConfigurationError(LifespanOnStartupError):
     msg_template = "Invalid postgres settings [={settings}] on startup. Note that postgres cannot be disabled using settings"
 
 
-def create_input_state(settings: PostgresSettings) -> State:
+def get_postgres_database_main_lifespan(settings: PostgresSettings) -> State:
```

Comment:
careful Andrei. This is not a lifespan (i.e. a (co)routine that gets app and state and returns an `Iterator[State]` ) .  This is the state expected by the `postgres_database_lifespan`